src/Event/EventLDProjector.php

```diff
@@ -296,6 +363,58 @@ protected function applyDescriptionTranslated(
     }
 
     /**
+     * Apply the description updated event to the event repository.
+     * @param DescriptionUpdated $descriptionUpdated
+     */
+    protected function applyDescriptionUpdated(
+        DescriptionUpdated $descriptionUpdated
+    ) {
+        $document = $this->loadDocumentFromRepository($descriptionUpdated);
+
+        $eventLd = $document->getBody();
+        $eventLd->description->{'nl'} = $descriptionUpdated->getDescription();
+
+        $this->repository->save($document->withBody($eventLd));
+    }
+
+    /**
+     * Apply the typical age range updated event to the event repository.
+     * @param TypicalAgeRangeUpdated $typicalAgeRangeUpdated
+     */
+    protected function applyTypicalAgeRangeUpdated(
+        TypicalAgeRangeUpdated $typicalAgeRangeUpdated
+    ) {
+        $document = $this->loadDocumentFromRepository($typicalAgeRangeUpdated);
+
+        $eventLd = $document->getBody();
+
+        if ($typicalAgeRangeUpdated->getTypicalAgeRange() === "-1") {
```

Comment:
$typicalAgeRangeUpdated doesn't capture the intention of the user well here. What the user has intended to do is - I guess - indicating that he/she does not know what's the typicalAgeRange for this event - or that it's suitable for all ages. It's better to introduce another type of event then for these cases. Setting the value to -1 feels to hacky and one needs too much implicit knowledge about the implementation of the system to know what it means.